@here/olp-sdk-dataservice-read/lib/client/CatalogEarliestVersionRequest.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { validateBillingTag } from "..";
+
+/**
+ * A class that prepare information about catalog earliest version.
+ */
+export class CatalogEarliestVersionRequest {
+    private earliestVersion?: number;
```

Comment:
Looking at the REST API I don't see any version you need to pass.
What is it used for?